dist/redhat/centos7-systemd.inc

```diff
@@ -0,0 +1,21 @@
+%undefine systemd_post
+%define systemd_post() \
```

Comment:
I think this comes from the original rpm files. We can't just copy without providing copyright and license information.

Since rpm is GPL/LGPL we can't use it directly, so we have to reimplement it without copying (it's okay to look at the source for inspiration).

Reply:
You are right, I just pushed re-implemented version. how about this? https://github.com/scylladb/scylla/pull/5605/files#diff-604df91d4cd4c6756e5c01e0e25fd324